.jscsrc

```diff
@@ -8,6 +8,7 @@
     "requireSpacesInAnonymousFunctionExpression": {
         "beforeOpeningCurlyBrace": true
     },
+    "requirePaddingNewLinesAfterBlocks": false,
```

Comment:
I think this was added by coins-validate (older version)? Anyway, BK-shield needs to be converted to eslint, I figured this pull was not the place to do so.